### PR TITLE
Add issueEventSourcePublisher

### PR DESCRIPTION
**Pull request by bigfluffycookie**
fixes https://github.com/SonarSource/sonarlint-visualstudio/issues/3788

### CODE REVIEW

File: src/ConnectedMode.UnitTests/ServerSentEvents/SSESessionTests.cs [hunk 1]
```diff
@@ -115,13 +120,15 @@ public async Task PumpAllAsync_WhenUnsupportedEvent_Ignores()
             new IServerEvent[]
             {
                 Mock.Of<ITaintVulnerabilityRaisedServerEvent>(),
-                Mock.Of<IIssueChangedServerEvent>(),
-                Mock.Of<ITaintVulnerabilityRaisedServerEvent>()
+                Mock.Of<IDummyServerEvent>(),
```
Comment (bigfluffycookie) [inline src/ConnectedMode.UnitTests/ServerSentEvents/SSESessionTests.cs:123]: Added a dummy server event to check if we ignores unsuppported events

File: src/ConnectedMode/Suppressions/IIssueServerEventSourcePublisher.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿/*
```
Comment (bigfluffycookie) [inline src/ConnectedMode/Suppressions/IIssueServerEventSourcePublisher.cs:1]: This is a copy of ITainIssueServerEventSourcePublisher.

File: src/ConnectedMode/ServerSentEvents/SSESessionFactory.cs [hunk 3]
```diff
@@ -22,6 +22,7 @@
 using System.ComponentModel.Composition;
```
Comment (bigfluffycookie) [inline src/ConnectedMode/ServerSentEvents/SSESessionFactory.cs:22]: Just copied what was already there for taint

**Review by duncanp-sonar — APPROVED** [review]
Nice

**Pull request merged**